python/sidecar/ai/schemas.py

```diff
@@ -61,6 +61,7 @@ class ChatRequest(RefStudioModel):
     text: str
     n_choices: int = 1
     temperature: float = 0.7
+    stream: bool = False
```

Comment:
Given that we will use a different endpoint, I think we don't need this in the request payload. 

With this approach maybe we can reuse the existing API endpoint `/api/ai/{project_id}/chat` and decide on the return type using that field.

Keeping this change and two endpoints, mean that we need to throw an error if someone call the `/api/ai/{project_id}/chat` endpoint with `stream: true`.

Reply:
How about `ChatRequest` remains the same (i.e. remove `stream` property), but `ChatRequest` is used by both endpoints?

Internally within `/chat_stream` I'll pass in `stream: true` as these two endpoints share much of the same code, but ultimately need to fork a slightly different path.

Reply:
Exactly. That's a good solution IMO.